cmd/commands/runtime.go

```diff
@@ -2696,14 +2696,31 @@ func postInstallationHandler(ctx context.Context, opts *RuntimeInstallOptions, e
 	if err != nil && !*disableRollback {
 		summaryArr = append(summaryArr, summaryLog{"----------Uninstalling runtime----------", Info})
 		log.G(ctx).Warnf("installation failed due to error : %s, performing installation rollback", err.Error())
+		iscCloneOpts := &git.CloneOptions{
+			FS:               fs.Create(memfs.New()),
+			CreateIfNotExist: false,
+		}
+		iscCloneOpts.Repo, err = getIscRepo(ctx)
+		handleCliStep(reporter.UninstallStepGetIscRepoFromPlatform, "Getting isc repo from platform before rollback", err, false, true)
+		if err != nil {
+			log.G(ctx).Errorf("failed to get isc repo from platform before installation rollback: %s", err.Error())
+		}
+
+		inferProviderFromRepo(opts.InsCloneOpts)
+		iscCloneOpts.Auth = opts.InsCloneOpts.Auth
+		iscCloneOpts.Progress = opts.InsCloneOpts.Progress
+		iscCloneOpts.Parse()
+		iscCloneOpts.Parse()
```

Comment:
seems like a copy-paste error. if it doesn't cause any problems (calling parse twice), and the fix is high priority to be released asap, you can keep it for now

Reply:
yep, copy paste error. will remove it